src/test/java/software/amazon/cloudformation/resource/ResourceTypeSchemaTest.java

```diff
@@ -101,6 +101,16 @@ public void getWriteOnlyProperties() {
         assertThat(result).containsExactly("/properties/propertyC", "/properties/propertyE/nestedProperty");
     }
 
+    @Test
+    public void getReplacementStrategy() {
+        JSONObject o = loadJSON(TEST_SCHEMA_PATH);
+        final ResourceTypeSchema schema = ResourceTypeSchema.load(o);
+
+        List<String> result = schema.getReplacementStrategy();
+        assertThat(result).containsExactly("create", "delete");
```

Comment:
can we have a test for the reverse as well? might be good for sanity to know both cases serialize correctly